src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs:React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id)
+  || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                key={id}
+                className={cn({
+                  'is-active': selectedTab.id === id,
+                })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (id !== selectedTabId) {
+                      onTabSelected(tab);
+                    }
+                  }}
+                >
+                  {title}
+                </a>
+              </li>
+            );
+          })}
```

Comment:
i think this would deserve it's own `.tsx` component file
It's simply looks kind of bulky right inside the return statement
Although it's not that big so we can live with that 😉